src/entities/trades/uniswap-v2/pairs/queries.ts

```diff
@@ -0,0 +1,34 @@
+import { gql } from 'graphql-request'
+
+export const GET_ALL_COMMON_PAIRS_BETWEEN_TOKEN_A_AND_TOKEN_B = gql`
+  query GetAllCommonPairsBetweenTokenAAndTokenB($tokenA: String!, $tokenB: String!) {
+    directPairs: pairs(where: { token0_in: [$tokenA, $tokenB], token1_in: [$tokenB, $tokenA] }) {
```

Comment:
Direct pairs are probably not required here since it's only one and it depends on whether `tokenA < tokenB`. In this case `tokenA` is `token0` in the pair and `tokenB` is `token1`, vice-versa otherwise. It's entirely derivable on the client side and probably shaves off a few milliseconds lol (negligible, but still)

Reply:
Removed in [7ea391c](https://github.com/levelkdev/swapr-sdk/pull/91/commits/7ea391c9f238d509b95c7b1308377c69cfd259bf)